docs/specs/helpers.md

```diff
@@ -44,6 +42,100 @@ The anchor point is highlighted with an orange dot.
     </script>
 </lottie-playground>
 
+Transforms the parent's coordinate system.
+
+When calculating a transform matrix, properties SHOULD be applied as follows:
+
+1. Translate by $-a$
+1. Scale by $\frac{s}{100}$
+1. If $sk \neq 0$:
+    1. Rotate by $-sa$
+    1. Skew x by $\tan(-sk)$
+    1. Rotate by $sa$
+1. Rotate by $-r$
+1. Translate by $p$
+
+Assuming a transform matrix with the following layout, with the labels equivalent to the
+[CSS matrix transform](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix):
+
+$$
+\begin{pmatrix}
+a & b & 0 & 0 \\
+c & d & 0 & 0 \\
+0 & 0 & 1 & 0 \\
+tx & ty & 0 & 1
```

Comment:
CSS calls them tx ty

Reply:
I see Mozilla mixes a/b/c/d/tx/ty in their docs, but the actual spec doesn't: https://drafts.csswg.org/css-transforms/#MatrixDefined

Usually the a,b,c,d,e,f notation is used to explain how a flattened representation (vector of floats) maps to the matrix coefficients (row-major vs. column major), while tx/ty shows up when talking about specific types of transforms (translation).

I think the "layout" depiction should stick to generic coefficients, and maybe add separate definitions for translation/scale/rotation/skew.

Or, since we don't have to deal with a flattened matrix representation, we could skip the general layout altogether and just define specific operations.